src/api/resources/collect_record.py

```diff
@@ -238,3 +230,40 @@ def missing(self, request, *args, **kwargs):
             missing_ids = [row[0] for row in cursor.fetchall()]
 
         return Response(dict(missing_ids=missing_ids))
+
+    @action(
+        detail=False, methods=["POST"], permission_classes=[ProjectDataAdminPermission]
+    )
+    def ingest(self, request, project_pk, *args, **kwargs):
+        supported_content_types = (
+            "application/csv",
+            "application/x-csv",
+            "text/csv",
+            "text/comma-separated-values",
+            "text/x-comma-separated-values",
+            "text/tab-separated-values",
```

Comment:
Are we sure we want this? This would mean no tabs in notes fields, basically, right?